### DISCUSSION TITLE
$event.dataTransfer.dropEffect = 'copy' not getting applied

### DISCUSSION THREAD

**Discussion by boustanihani** [alpinejs/alpine#4443] [7. Bugs (archived)]
I have an area where I can drag and drop files to:
```
<div @dragover.prevent=" $event.dataTransfer.dropEffect = 'copy' "
    @drop.prevent=" filesSelected($event.dataTransfer.files) "
    @change.prevent=" filesSelected($event.target.files) ">
    <input type="file" id="files" name="files[]" multiple />
</div>
```

The `drop` and the `change` events are getting fired as they should, only the `dropEffect = 'copy'` is not getting applied.

**Comment by ekwoka** [comment]
How do you know it's not being applied?

Here I don't see any thing that would change based on that value.

So what makes you think it's not being applied?

**Comment by boustanihani** [comment]
It was getting applied, I just thought something is wrong because on windows you also see the number of the files (above the mouse cursor), but on linux you don't..
